Formula/netdiscover.rb

```diff
@@ -0,0 +1,30 @@
+class Netdiscover < Formula
+  desc "Active/passive ARP reconnaissance tool"
+  homepage "https://github.com/netdiscover-scanner/netdiscover"
+  url "https://github.com/netdiscover-scanner/netdiscover/archive/refs/tags/0.9.tar.gz"
+  sha256 "dc436992cae0e61ad8d14e1fb4d2650ec9e754c0024010e15d341afb98e28f70"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```

Comment:
```suggestion
  depends_on "automake" => :build
  uses_from_macos "libpcap"
```
Seems like you need this.